application/src/main/java/be/jpendel/application/PersonMapper.java

```diff
@@ -6,7 +6,9 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class PersonMapper {
+/* TODO : mark: PersonMapper en PersonDTO moeten volgens mij in de domain layer
+*/
+class PersonMapper {
```

Comment:
Ok, dus de applicatie laag gaat gewoon echt alles gaan 'gebruiken' uit het domein. Het domein is niet echt een laag zoals ik dat zie.

Maar wat dan wel echt mis is is het feit dat in al die factories een dependency ligt naar de Commands in application. Ofwel moeten die Commands naar binnen, ofwel moeten ze een interface implementeren die in het domein zit.